src/services/items.js

```diff
@@ -1,12 +1,19 @@
 import axios from 'axios'
 
 const baseUrl = 'https://gp-super-store-api.herokuapp.com/'
+const PAGES_SIZE = 6
```

Comment:
Since you use this same constant in many places, it's best to store it in a file at `/src/utils/constants` and import it whenever you need it. That makes it easier to change if you ever need to update this value.